### PR TITLE
Date Repaid field

### PR DESCRIPTION
**Pull request by jaivigneshvenugopal**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/DateRepaid.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package seedu.address.model.person;
+
+/**
+ * Represents the date of when the Person was sent to the whitelist in the address book, i.e. the date
+ * the Person repays his debt completely.
+ * Guarantees: immutable;
+ */
+public class DateRepaid {
+
+    public static final String STANDARDISED_PLACEHOLDER = "NOT REPAID";
```
Comment (khooroko) [inline src/main/java/seedu/address/model/person/DateRepaid.java]: name is not developer-intuitive enough. consider "DATE_REPAID_PLACEHOLDER" or "MESSAGE_NOT_REPAID" or other names along those lines

**Comment by khooroko** [comment]
I think you forgot to update InfoPanelTest.java

File: src/main/java/seedu/address/model/person/DateRepaid.java (hunk 1 above)
Comment (jaivigneshvenugopal) [reply src/main/java/seedu/address/model/person/DateRepaid.java]: Noted. Thanks for the suggestion!

**Review by khooroko — APPROVED** [review]
good job

File: src/test/java/guitests/guihandles/InfoPanelHandle.java [hunk 2]
```diff
@@ -100,9 +107,9 @@ public String getDebt() {
         return debtLabel.getText();
     }
 
-    public String getDateBorrow() {
-        return dateBorrowLabel.getText();
-    }
+    public String getDateBorrow() { return dateBorrowLabel.getText(); }
+
+    public String getDateRepaid() { return dateRepaidLabel.getText(); }
```
Comment (khooroko) [inline src/test/java/guitests/guihandles/InfoPanelHandle.java]: how did these pass Travis
anw it's not supposed to be single line. same thing in PersonCardHandle.java

Comment (jaivigneshvenugopal) [reply src/test/java/guitests/guihandles/InfoPanelHandle.java]: Noted. Made the necessary changes.

Comment (jaivigneshvenugopal) [reply src/test/java/guitests/guihandles/InfoPanelHandle.java]: Noted. Made the necessary changes.

**Review by jelneo — APPROVED** [review]
nice!

File: src/main/java/seedu/address/model/person/DateRepaid.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.person;
+
+/**
+ * Represents the date of when the Person was sent to the whitelist in the address book, i.e. the date
+ * the Person repaid his debt completely.
+ * Guarantees: immutable;
+ */
+public class DateRepaid {
+
+    public static final String DATE_REPAID_PLACEHOLDER = "NOT REPAID";
```
Comment (lawwman) [inline src/main/java/seedu/address/model/person/DateRepaid.java:10]: Refer to my pull request that added in Date Borrow field. That should give you an idea of how to use the Date class.

**Review by lawwman — APPROVED** [review]
Good job implementing the new field.
Be sure to test it when command to send person to a Repaid List is made.

**Pull request merged**